README.md

```diff
@@ -67,6 +67,19 @@ software](clang/docs/checkedc/3C/README.md), including development status and
 how to contribute. Here are the [usage instructions for the `3c` command-line
 tool](clang/tools/3c/README.md).
 
+## Quick Setup 
```

Comment:
I don't think adding this to the root readme is going to be acceptable to Microsoft.  Thus, if you want to do it, we need to first decide on a solution to diverge the repositories (#302), though I'm still skeptical that it's a net positive to duplicate the information.  Or we could look into ways to make the 3C installation instructions easier for our users to find without changing the root readme.

Reply:
I think we need some way to help new users find this information. As it is it's 4 levels deep in folders, and requires having some intuition about the repository layout.

Reply:
From our meeting earlier: Currently we are still synchronizing this repository with Microsoft's, and just because we think content is important enough to add to the root readme doesn't mean we can convince Microsoft, though we can negotiate with them.  But one change that might be sufficient to solve this problem and that I expect Microsoft would accept is to add a direct link from the previous section (about 3C) to `clang/docs/checkedc/3C/INSTALL.md` and maybe reformat the three links as bullet points to make them easier to read.

Reply:
Here's [my draft revision of the 3C section](https://github.com/correctcomputation/checkedc-clang/compare/main...docupdates-matt).  If you like this, you can copy the text into this PR, or I can submit a separate PR, or whatever.